### PR TITLE
BraintreeGraphQl Workflow

### PR DESCRIPTION
**Pull request by pmclain**
## Purpose of this pull request

Corresponds with magento/graphql-ce#392

This PR adds workflow documentation for queries and mutations required for setting quote payment methods to Braintree and Braintree CC Vault.

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.3/graphql/payment-methods/braintree.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+### braintree object
+
+The `braintree` object must contain the following attributes:
+
+Attribute |  Data Type | Description
+--- | --- | ---
+`payment_method_nonce` | String! | Required input for Braintree client-side generated nonce
+`is_active_payment_token_enabler` | Boolean! | Required input dictating if payment should be stored in `Magento_Vault`
+`device_data` | String | Optional input json encoded device data for Kount integration
```
Comment (keharper) [inline guides/v2.3/graphql/payment-methods/braintree.md]:
```suggestion
`device_data` | String | Optional. JSON-encoded device data for Kount integration
```

**Comment by keharper** [comment]
running tests

**Comment by keharper** [comment]
Thanks @pmclain for this major contribution. I'll contact you within a couple of days to shore up the infographics for the workflows.

**Pull request merged**